tools/docker_dev/.gitpod.Dockerfile

```diff
@@ -0,0 +1,79 @@
+# Using the Scipy-dev Docker image as a base
+# This way, we ensure we have all the needed compilers and dependencies
+# while reducing the build time
+ARG BASE_CONTAINER=trallard/scipy:scipy-dev.4
+FROM ${BASE_CONTAINER}
+
+# -----------------------------------------------------------------------------
+# ---- OS dependencies ----
+# hadolint ignore=DL3008
+RUN apt-get update && \ 
+    apt-get install -yq --no-install-recommends \
+    zip \
+    unzip \
+    bash-completion \
+    build-essential \
+    htop \
+    jq \
+    less \
+    locales \
+    man-db \
+    sudo \
+    time \
+    multitail \
+    lsof \
+    ssl-cert \
+    && locale-gen en_US.UTF-8 \
+    && apt-get clean \
+    && rm -rf /var/cache/apt/* \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /tmp/*
+
+RUN add-apt-repository -y ppa:git-core/ppa \
+    && apt-get install -yq --no-install-recommends git \
+    && rm -rf /var/lib/apt/lists/* 
```

Comment:
ok, this will add git into the container, correct? 

@trallard I had left a comment over [here](https://github.com/scipy/scipy/pull/13447#issuecomment-776413118)

about not having `git` available inside the container when I setup-this looks like it fixes that, thanks!

Reply:
Is there a reason this can't be done in the previous run command that does a whole load of apt-get installs?

Reply:
Thanks @rlucas7 I wonder if it would be worth adding git into the other image actually for those that would be using Docker.

@andyfaff  no there is not ... all the apt-get installs can be done together. I was planning to do some tidying up of the Dockerfile and other configs soon as this is still quite WIP

Reply:
> Thanks @rlucas7 I wonder if it would be worth adding git into the other image actually for those that would be using Docker.

Ah I see now that this file is actually different from the other file for the docker container-thanks for pointing out. I'll open a PR to add.

Reply:
Fab - will then change this Dockerfile as it uses the one being updated in #13551